src/Store.ts

```diff
@@ -32,14 +34,14 @@ export let defaultStoreName = "<Anonymous-Store>";
  * Store class
  * @public
  */
-export default class Store extends Dispatcher {
+abstract class Store extends Dispatcher {
```

Comment:
> Is it valid in TypeScript?

"At this time", it's valid in TypeScript because we keep all unit test cases as plain js.
However, of course, we need to add `class TestStore extends Store` to them if we convert test cases to TypeScript.